src/SprykerEco/Yves/Computop/Handler/PostPlace/ComputopDirectDebitPaymentHandler.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * MIT License
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerEco\Yves\Computop\Handler\PostPlace;
+
+use Generated\Shared\Transfer\ComputopDirectDebitPaymentTransfer;
+use Generated\Shared\Transfer\QuoteTransfer;
+use Spryker\Shared\Kernel\Transfer\AbstractTransfer;
+
+class ComputopDirectDebitPaymentHandler extends AbstractPostPlacePaymentHandler
+{
+    /**
+     * @param \Generated\Shared\Transfer\QuoteTransfer $quoteTransfer
+     * @param \Spryker\Shared\Kernel\Transfer\AbstractTransfer $responseTransfer
+     *
+     * @return \Generated\Shared\Transfer\QuoteTransfer
+     */
+    protected function addPaymentToQuote(QuoteTransfer $quoteTransfer, AbstractTransfer $responseTransfer)
+    {
+        if ($quoteTransfer->getPayment()->getComputopDirectDebit() === null) {
```

Comment:
From what I've learned in code, this IF is not required.
It will be always TRUE.